.github/workflows/pr-ci.yaml

```diff
@@ -276,6 +276,9 @@ jobs:
   auto-commit-schemas:
     needs: [compare-graphql-schema, compare-openapi-schema, checkout-branch, update-schemas]
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
```

Comment:
Is this needed? Should it be a separate PR?

Reply:
Yes, I will open a separate PR for this.